docs/package-sign.md

```diff
@@ -20,7 +20,19 @@ Make sure you have the following:
 
 -   [Git](https://git-scm.com/downloads)
 
- 
+### Ensure your connector is ready to be packaged
+
+As part of the packaging process, the packager will ensure that:
+
+-   All xml files are valid against the XSD files located in the validate folder of the SDK
+
+-   All files referenced exist
+
+-   All files are smaller than 3 mb
+
+-   The connection class is the same in all places
+
+However, this validation does not guarantee that the connector will work in Tableau. The packager does not validate javascript, for example, or that the connection class is unique. To ensure that your connector works before packaging it you can [run your unpackaged connector]({{ site.baseurl }}/docs/share), or test it using [TDVT]({{ site.baseurl }}/docs/tdvt). 
```

Comment:
Replace javascript with JavaScript.
Add a comma: To ensure that your connector works before packaging it, you can...